.devcontainer/Dockerfile

```diff
@@ -344,7 +344,7 @@ RUN ${PYTHON_VENV_EXECUTABLE} -m pip install --quiet --upgrade --no-cache-dir "s
 ### install python3-aioeventlet and bcc-tools from the magma apt repo
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/
 RUN apt-key add /tmp/jfrog.pub && \
```

Comment:
Does it make sense to use here (and in other Dockerfiles where possible)
`RUN wget -qO - https://artifactory.magmacore.org:443/artifactory/api/gpg/key/public | apt-key add -`
so that the jobs in this PR are not dependent on the not yet updated `jfrog.pub`.

Reply:
It makes sense, honestly I was just lazy and just copy pasted, it was painful to go through all the sources haha